packages/wp-now/src/run-cli.ts

```diff
@@ -87,22 +90,25 @@ export async function runCli() {
 			}
 		)
 		.command(
-			'php <file>',
-			'Run the php command passing the arguments for php cli',
+			'php [..args]',
+			'Run the php command passing the arguments to php cli',
 			(yargs) => {
 				commonParameters(yargs);
-				yargs.positional('file', {
-					describe: 'Path to the PHP file to run',
-					type: 'string',
-				});
+				yargs.strict(false);
 			},
 			async (argv) => {
 				try {
+					// 0: node, 1: wp-now, 2: php, ...args
+					const args = process.argv.slice(2);
 					const options = await getWpNowConfig({
 						path: argv.path as string,
 						php: argv.php as SupportedPHPVersion,
 					});
-					await executePHPFile(argv.file as string, options);
+					const phpArgs = args.includes('--')
+						? (argv._ as string[])
+						: args;
+					// 0: php, ...args
```

Comment:
Would it be worthwhile to have some tests for this `args` parsing? The bummer is that this isn't quite covered with our test coverage.